utils/build.ps1

```diff
@@ -103,7 +106,8 @@ param(
   [string[]] $Test = @(),
   [string] $Stage = "",
   [string] $BuildTo = "",
-  [switch] $ToBatch
+  [switch] $ToBatch,
+  [string] $PinnedLayout = ""
```

Comment:
Can we default this to `old` instead?  Or perhaps we should use `NewPinnedLayout`?

Reply:
Done